app/src/main/java/com/launcher/silverfish/launcher/LauncherActivity.java

```diff
@@ -183,29 +180,6 @@ public void setFragShortcutAddListenerRefreshListener(ShortcutAddListener shortc
 
     //region Events
 
-    private void dropItem(DragEvent dragEvent) {
```

Comment:
Why did you remove this code?

Reply:
Android studio said the method is never used. Shall I add it back?

Reply:
Roger that :+1:

Reply:
I removed that commit and force pushed it. Did that work?